sim/midas/src/main/scala/midas/passes/MidasTransforms.scala

```diff
@@ -44,18 +45,32 @@ private[midas] class MidasTransforms(implicit p: Parameters) extends Transform {
       firrtl.passes.SplitExpressions,
       firrtl.passes.CommonSubexpressionElimination,
       new firrtl.transforms.DeadCodeElimination,
-      new EnsureNoTargetIO,
-      // NB: Carelessly removing this pass will break the FireSim manager as we always
-      // need to generate the *.asserts file. Fix by baking into driver.
+      EnsureNoTargetIO,
+      new BridgeExtraction,
+      new ResolveAndCheck,
+      new EmitFirrtl("post-bridge-extraction.fir"),
+      new HighFirrtlToMiddleFirrtl,
+      new MiddleFirrtlToLowFirrtl,
+      new AutoCounterTransform,
+      new ResolveAndCheck,
       new AssertPass(dir),
       new PrintSynthesis(dir),
       new ResolveAndCheck,
+      TriggerWiring,
+      new EmitFirrtl("post-trigger-wiring.fir"),
+      new fame.EmitFAMEAnnotations("post-trigger-wiring.json"),
+      new ResolveAndCheck,
       new HighFirrtlToMiddleFirrtl,
-      new MiddleFirrtlToLowFirrtl,
-      new BridgeExtraction,
+      new WiringTransform,
       new ResolveAndCheck,
+      new HighFirrtlToMiddleFirrtl,
       new MiddleFirrtlToLowFirrtl,
-      new fame.WrapTop,
+      // We should consider moving these lower
+      ChannelClockInfoAnalysis,
+      UpdateBridgeClockInfo,
+      new EmitFirrtl("post-debug-synthesis.fir"),
+      new fame.EmitFAMEAnnotations("post-debug-synthesis.json"),
```

Comment:
I think leaving these json outputs in is good, since I switched them to dump only the FAME annotations.

Reply:
Cool.